webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/reports/ReportingServiceImpl.scala

```diff
@@ -96,6 +97,7 @@ object CacheComplianceQueueAction {
   final case class RemoveNodeInCache      (nodeId: NodeId                                          ) extends CacheComplianceQueueAction
   final case class InitializeCompliance   (nodeId: NodeId, nodeCompliance: Option[NodeStatusReport]) extends CacheComplianceQueueAction // do we need this?
   final case class UpdateCompliance       (nodeId: NodeId, nodeCompliance: NodeStatusReport        ) extends CacheComplianceQueueAction
+  final case class InitNodeConfiguration  (nodeId: NodeId, nodeConfiguration: NodeExpectedReports  ) // unsure about keeping this one
```

Comment:
it's unused, i' pretty sure i should not keep it

Reply:
clean it, it's cognitively more costly to have something and wonder why it's there than needing to add it if needed later ont